post_installation_configuration/aws-compute-edge-tasks-zones.adoc

```diff
@@ -1,50 +1,51 @@
 :_mod-docs-content-type: ASSEMBLY
-[id="aws-compute-edge-tasks"]
-= AWS Local Zone tasks
+[id="aws-compute-edge-tasks-zones"]
+= AWS zone tasks
 include::_attributes/common-attributes.adoc[]
-:context: aws-compute-edge-tasks
+:context: aws-compute-edge-tasks-zones
+:zone-type: Local Zones or Wavelength Zones
 
 toc::[]
 
-After installing {product-title} on Amazon Web Services (AWS), you can further configure AWS Local Zones and an edge compute pool, so that you can expand and customize your cluster to meet your needs.
+After installing {product-title} on Amazon Web Services (AWS), you can further configure AWS {zone-type} and an edge compute pool, so that you can expand and customize your cluster to meet your needs.
 
-// Extend existing clusters to use AWS Local Zones
+// Extend existing clusters to use AWS zones
 include::modules/post-install-edge-aws-extend-cluster.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
 
 * For more information about AWS Local Zones, the supported instances types, and services, see link:https://aws.amazon.com/about-aws/global-infrastructure/localzones/features/[AWS Local Zones features] in the AWS documentation.
+* For more information about AWS Local Zones, the supported instances types, and services, see link:https://aws.amazon.com/wavelength/features/[AWS Wavelength features] in the AWS documentation.
 
-
-// About the Edge Compute Pool
+// Edge compute pools and AWS zones
 include::modules/edge-machine-pools-aws-local-zones.adoc[leveloffset=+2]
 
 [id="post-install-extend-existing-to-local-zones-mtu"]
-=== Changing the Cluster Network MTU to support AWS Local Zones subnets
+=== Changing the Cluster Network MTU to support AWS zone subnets
 
 You might need to change the maximum transmission unit (MTU) value for the cluster
-network so that your cluster infrastructure can support Local Zone subnets.
+network so that your cluster infrastructure can support {zone-type} subnets.
 
 // About the cluster MTU
 include::modules/nw-cluster-mtu-change-about.adoc[leveloffset=+3]
 
-// Changing the cluster MTU
+// Changing the cluster MTU -- CONFIRM WITH MARCO as steps differ
 include::modules/nw-cluster-mtu-change.adoc[leveloffset=+3]
 
-// Opting in to AWS Local Zones
-include::modules/installation-aws-add-local-zone-locations.adoc[leveloffset=+2]
+// Opting in to an AWS zone
+include::modules/installation-aws-add-zone-locations.adoc[leveloffset=+2]
 
-// Extend existing clusters to use AWS Local Zones
-include::modules/post-install-existing-local-zone-subnet.adoc[leveloffset=+2]
+// Create network requirements in an existing VPC to use an AWS zone
+include::modules/post-install-extend-existing-to-zone-type.adoc[leveloffset=+2]
 
-// Creating a subnet in AWS Local Zones
-include::modules/installation-creating-aws-subnet-localzone.adoc[leveloffset=+3]
```

Comment:
@dfitzmau there are missing two modules when you migrated from original PR, task book, to here.

There are a section describing to create carrier gateways when setting up WL in existing VPC, two new modules introduced in:
https://github.com/openshift/openshift-docs/pull/69128/files#diff-2144a9c5b151f2651e449394a6d29ec6be8da1b2c190f62eeff27777305b0b38R78-R100

Please review it, and it is important to note that it is valid only for WL Zones.

Reply:
Thanks. Updated. I added the following modules to the AWS edge zone tasks doc book:

// Creating a VPC carrier gateway
include::modules/installation-creating-aws-vpc-carrier-gw.adoc[leveloffset=+2]

// CloudFormation template for the VPC Carrier Gateway
include::modules/installation-cloudformation-vpc-carrier-gw.adoc[leveloffset=+2]